.env

```diff
@@ -13,3 +13,6 @@ SPRINGBOARD_BUCKET=mbta-ctd-dataplatform-dev-springboard
 ARCHIVE_BUCKET=mbta-ctd-dataplatform-dev-archive
 ERROR_BUCKET=mbta-ctd-dataplatform-dev-error
 INCOMING_BUCKET=mbta-ctd-dataplatform-dev-incoming
+
+# api env stuff
+MIX_ENV=dev
```

Comment:
if we're running locally, i think we want the dev environment.

Reply:
This is the default, so you don't need to include it in the environment variables.